packages/electrode-archetype-react-app-dev/config/webpack/add-dll-references.js

```diff
@@ -5,6 +5,17 @@ const fs = require("fs");
 const glob = require("glob");
 const webpack = require("webpack");
 const archetype = require("../archetype");
+const winston = require("winston");
+const logger = new (winston.Logger)({
+  transports: [
+    new (winston.transports.Console)({
+      colorize: true
+    }),
+    new (winston.transports.File)({
+      filename: 'winston.log'
+    })
+  ]
+});
```

Comment:
Not sure about `confippet` (I think I haven't used it yet), but definitely somewhere configurable.